pkg/operator/certrotation/cabundle.go

```diff
@@ -58,16 +58,25 @@ func (c CABundleConfigMap) EnsureConfigMapCABundle(ctx context.Context, signingC
 		)}
 	}
 
+	isPrimaryOwner := false
 	needsMetadataUpdate := false
 	if c.Owner != nil {
 		needsMetadataUpdate = ensureOwnerReference(&caBundleConfigMap.ObjectMeta, c.Owner)
+		if caBundleConfigMap.ObjectMeta.OwnerReferences[0] == *c.Owner {
```

Comment:
Why can't we just have a single owner?

Reply:
CA bundle can contain multiple signers, controlled by different controllers. We could have a single ownerRef, but that wouldn't reflect reality. It also makes easier to find such CA bundles in the must-gathers

Reply:
> CA bundle can contain multiple signers, controlled by different controllers

This is wrong and should be fixed. Why can't we have a single signer ?
Could you point me to code that does this ?

Reply:
Still, there is a single signer and a CA bundle. 
Also as far as I can tell `kas-o` doesn't make use of the `Owner` field for both signer and CA resources.

Reply:
I don't understand why we have to have multiple controllers controlling a single resource. I would prefer to have a single controller for that.

It seems to me that we would like to mark which components are using the resource, which could be done with a single controller.